homeassistant/core.py

```diff
@@ -864,6 +865,31 @@ def async_register(self, domain, service, service_func, description=None,
             {ATTR_DOMAIN: domain, ATTR_SERVICE: service}
         )
 
+    def remove(self, domain, service):
+        """Remove a registered service from service handler."""
+        run_callback_threadsafe(
+            self._hass.loop, self.async_remove, domain, service).result()
+
+    @callback
+    def async_remove(self, domain, service):
+        """Remove a registered service from service handler.
+
+        This method must be run in the event loop.
+        """
+        domain = domain.lower()
+        service = service.lower()
+
+        if service not in self._services.get(domain, {}):
+            _LOGGER.error("No service %s fround for %s.", service, domain)
```

Comment:
`fround` should be `found`. But I think I rather prefer "Unable to remove unknown service %s/%s.". Also, make it a warning since it's not an error that should cause trouble.